docs/content/dagster-cloud/deployment/agents/docker.mdx

```diff
@@ -18,7 +18,7 @@ To complete the steps in this guide, you'll need:
 
 - **Permissions in Dagster Cloud that allow you to manage agent tokens**. Refer to the [User permissions documentation](/dagster-cloud/account/managing-users) for more info.
 - **To have Docker installed**
-- **Access to a container registry to which you can push images with Dagster code.** Additionally, your Docker agent must have the permissions required to pull images from the registry.
+- **Access to a container registry to which you can push images with Dagster code.** Additionally, your Docker agent must have the permissions required to pull images from the registry. See [below](#granting-registry-access-to-docker-agent) for more details.
```

Comment:
```suggestion
- **Access to a container registry to which you can push images with Dagster code.** Additionally, your Docker agent must have the permissions required to pull images from the registry. See [below](#step-3-grant-registry-access-to-the-agent) for more details.
```

This fixes a broken link, updating it to use the new name of the section heading.